### ISSUE TITLE
Build error when used with github-metadata

### ISSUE THREAD

**Issue by neerajgangwar** [inukshuk/jekyll-scholar#346]
Hi,

The Jekyll build on Github fails if `github-metadata` is used along with `jekyll-scholar`. I faced the following error in the Github Actions build:
```
/home/runner/work/neerajgangwar.github.io/neerajgangwar.github.io/vendor/bundle/ruby/3.1.0/gems/jekyll-scholar-7.1.0/lib/jekyll/scholar/utilities.rb:619:in `join': no implicit conversion of Jekyll::GitHubMetadata::Value into String (TypeError)
	from /home/runner/work/neerajgangwar.github.io/neerajgangwar.github.io/vendor/bundle/ruby/3.1.0/gems/jekyll-scholar-7.1.0/lib/jekyll/scholar/utilities.rb:619:in `details_link_for'
	from /home/runner/work/neerajgangwar.github.io/neerajgangwar.github.io/vendor/bundle/ruby/3.1.0/gems/jekyll-scholar-7.1.0/lib/jekyll/scholar/utilities.rb:531:in `bibliography_tag'
	from /home/runner/work/neerajgangwar.github.io/neerajgangwar.github.io/vendor/bundle/ruby/3.1.0/gems/jekyll-scholar-7.1.0/lib/jekyll/scholar/tags/bibliography.rb:62:in `block in render_items'
	from /home/runner/work/neerajgangwar.github.io/neerajgangwar.github.io/vendor/bundle/ruby/3.1.0/gems/jekyll-scholar-7.1.0/lib/jekyll/scholar/tags/bibliography.rb:61:in `each'
	from /home/runner/work/neerajgangwar.github.io/neerajgangwar.github.io/vendor/bundle/ruby/3.1.0/gems/jekyll-scholar-7.1.0/lib/jekyll/scholar/tags/bibliography.rb:61:in `each_with_index'
	from /home/runner/work/neerajgangwar.github.io/neerajgangwar.github.io/vendor/bundle/ruby/3.1.0/gems/jekyll-scholar-7.1.0/lib/jekyll/scholar/tags/bibliography.rb:61:in `each'
	from /home/runner/work/neerajgangwar.github.io/neerajgangwar.github.io/vendor/bundle/ruby/3.1.0/gems/jekyll-scholar-7.1.0/lib/jekyll/scholar/tags/bibliography.rb:61:in `map'
	from /home/runner/work/neerajgangwar.github.io/neerajgangwar.github.io/vendor/bundle/ruby/3.1.0/gems/jekyll-scholar-7.1.0/lib/jekyll/scholar/tags/bibliography.rb:61:in `render_items'
	from /home/runner/work/neerajgangwar.github.io/neerajgangwar.github.io/vendor/bundle/ruby/3.1.0/gems/jekyll-scholar-7.1.0/lib/jekyll/scholar/tags/bibliography.rb:[27](https://github.com/neerajgangwar/neerajgangwar.github.io/actions/runs/3252438369/jobs/5338649152#step:5:28):in `render'
	from /home/runner/work/neerajgangwar.github.io/neerajgangwar.github.io/vendor/bundle/ruby/3.1.0/gems/liquid-4.0.3/lib/liquid/block_body.rb:103:in `render_node_to_output'
```

The full build logs are available [here](https://github.com/neerajgangwar/neerajgangwar.github.io/actions/runs/3252438369/jobs/5338649152). This issue seems to be similar to [this issue](https://github.com/jekyll/github-metadata/issues/22), faced while using `github-metadata` and `jekyll-redirect-from` together. I can confirm that the build was successful after removing `github-metadata` from the dependency list and passing `url` explicitly in `_config.yml`.

Thank you in advance!

**Comment by inukshuk** [comment]
Thanks! This should be fixed in 7.1.1.

**Comment by neerajgangwar** [comment]
Thank you!
